src/main/java/uk/gov/hmcts/reform/fees/client/model/FeeOutcome.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fees.client.model;
+
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Data
```

Comment:
Can we please add a `@Builder` annotation? So that it's easier to construct these objects when adding tests on claim store side.
Ideally, all of the models might have a builder, but I currently just see `FeeOutcome` as useful :D